server/task_check_scheduler.go

```diff
@@ -347,6 +365,43 @@ func (s *TaskCheckScheduler) ScheduleCheckIfNeeded(ctx context.Context, task *ap
 	return task, nil
 }
 
+// createTaskCheckRunIfNeededBySchemaReviewPolicy gets the corresponding SchemaReviewPolicy
+// and generates specific TaskCheckRuns based on it.
+func (s *Server) createTaskCheckRunIfNeededBySchemaReviewPolicy(
+	ctx context.Context,
+	creatotID int,
+	taskID int,
+	environmentID int,
+	basePayload api.TaskCheckDatabaseStatementAdvisePayload,
+	skipIfAlreadyTerminated bool,
+) error {
+	policy, err := s.store.GetSchemaReviewPolicyByEnvID(ctx, environmentID)
+	if err != nil {
+		return err
+	}
+	for _, rule := range policy.RuleList {
+		if rule.Level == api.SchemaRuleLevelDisabled {
+			continue
+		}
+		taskCheckType, payload, err := getTaskCheckTypeAndPayloadByRule(rule, basePayload)
```

Comment:
I am thinking of only having a single task check type for all schema review rules.

Otherwise. there are just too many task checks (and each will surface on the UI).